mcchris/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang='en'>
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		
+		<title>mcchris</title>
+
+		<style>
+			body {
+				background-image: url(./img/bg.jpg);
+				color:white;
+				font-family: monospace;
+				margin: 6rem 1rem;
+				font-size: 120%;
+			}
+			h1, h2, h3 {
+				text-align: center;
+			}
+			.flex {
+				display: flex;
+				flex-wrap: wrap;
+				justify-content: space-evenly;
+				margin: 3rem 0;
+			}
+			a img {
+				height: 6rem;
+				width: 6rem;
+				margin: .5rem 0;
+				background-color: white;
+			}
+			a {
+
+			}
+		</style>
+	</head>
+	<body id='home'>
```

Comment:
The id `home` is not styled or used at all. This also doesn't appear to be the home page.